src/components/ReportViewer/log-table.js

```diff
@@ -114,6 +126,18 @@ const LogTable = ({ logs, onRowClick }) => {
         setSelectedRowIndex(-1);
     }, [logs]);
 
+    const filter = useCallback(
+        (row) => {
+            return (
+                row.severity &&
+                Object.entries(selectedSeverity).some(
+                    ([key, value]) => key === row.severity && value
+                )
```

Comment:
we are filtering rows which has falsy severity or false value
it was not done before, is it necessary ?

Reply:
After some investigation I've found some reports without severity in the db but those are dedicated to network metadata so they aren't linked to a node thus they wouldn't appear in the report viewer